src/sentry/models/group.py

```diff
@@ -459,7 +459,16 @@ def update_group_status(
                 if priority and group.priority != priority:
                     group.priority = priority
                     updated_priority[group.id] = priority
-
+            else:
+                logger.info(
+                    "group.update_group_status.priority_not_updated",
+                    extra={
+                        "group_id": group.id,
+                        "from_substatus": from_substatus,
```

Comment:
@snigdhas should we also put in the project id in the logs?

Reply:
the behavior shouldn't change between projects - group.id is sufficient